src/main/java/com/example/tour/rating/Rating.java

```diff
@@ -0,0 +1,63 @@
+package com.example.tour.rating;
+
+import com.example.tour.activity.Activity;
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Getter
+@Setter
+@Data
```

Comment:
@Data is a convenient shortcut annotation that bundles the features of @ToString, @EqualsAndHashCode, @Getter / @Setter and @RequiredArgsConstructor together

https://projectlombok.org/features/Data